sentinel-core/src/test/java/com/alibaba/csp/sentinel/eagleeye/EagleEyeCoreUtilsTest.java

```diff
@@ -0,0 +1,180 @@
+package com.alibaba.csp.sentinel.eagleeye;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class EagleEyeCoreUtilsTest {
+
+    @Rule
+    public final ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testIsBlank() {
+        Assert.assertTrue(EagleEyeCoreUtils.isBlank(""));
+        Assert.assertTrue(EagleEyeCoreUtils.isBlank(" "));
+        Assert.assertTrue(EagleEyeCoreUtils.isBlank(null));
+
+        Assert.assertFalse(EagleEyeCoreUtils.isBlank("foo"));
+    }
+
+    @Test
+    public void testCheckNotNullEmpty() {
+        Assert.assertEquals("foo",
+                EagleEyeCoreUtils.checkNotNullEmpty("foo", "bar"));
+    }
+
+    @Test
+    public void testCheckNotNullEmptyInputEmptyStringThrowsException() {
+        thrown.expect(IllegalArgumentException.class);
+        EagleEyeCoreUtils.checkNotNullEmpty("", "bar");
+        // Methods are not expected to return due to exception thrown
+    }
+
+    @Test
+    public void testCheckNotNullEmptyIntputSpaceThrowsException() {
+        thrown.expect(IllegalArgumentException.class);
+        EagleEyeCoreUtils.checkNotNullEmpty(" ", "bar");
+        // Methods are not expected to return due to exception thrown
+    }
+
+    @Test
+    public void testCheckNotNullEmptyInputNullThrowsException() {
+        thrown.expect(IllegalArgumentException.class);
+        EagleEyeCoreUtils.checkNotNullEmpty(null, "bar");
+        // Methods are not expected to return due to exception thrown
+    }
+
+    @Test
+    public void testCheckNotNull() {
+        Assert.assertEquals("", EagleEyeCoreUtils.checkNotNull("", "bar"));
+        Assert.assertEquals(" ", EagleEyeCoreUtils.checkNotNull(" ", "bar"));
+        Assert.assertEquals("foo",
+                EagleEyeCoreUtils.checkNotNull("foo", "bar"));
+
+        thrown.expect(IllegalArgumentException.class);
+        EagleEyeCoreUtils.checkNotNull(null, "bar");
+        // Method is not expected to return due to exception thrown
+    }
+
+    @Test
```

Comment:
This looks like the same exception being checked as in the previous test method, testCheckNotNull().
I think either is ok but tend towards testing exceptions separately.

Reply:
Oops forgot to delete that